src/main.rs

```diff
@@ -708,10 +708,25 @@ impl TypeDarwin {
 //         line = line.replace('\\ ', ' ')
 //         line = line.replace('\& ', '')
 //         return line
+#[test]
+fn test_TypeDarwin_groff_replace_escapes() {
+    // tests for expected replacements
+    assert!(TypeDarwin::groff_replace_escapes(".Nm") == "CMDNAME");
+    assert!(TypeDarwin::groff_replace_escapes(r"\\ ") == " ");
+    assert!(TypeDarwin::groff_replace_escapes(r"\& ") == "");
+    // tests for no expected replacement
+    assert!(TypeDarwin::groff_replace_escapes(".N") == ".N");
+    assert!(TypeDarwin::groff_replace_escapes(r"\ ") == r"\ ");
+    assert!(TypeDarwin::groff_replace_escapes(r"\\") == r"\\");
+    assert!(TypeDarwin::groff_replace_escapes(r"& ") == "& ");
+    assert!(TypeDarwin::groff_replace_escapes(r"\&") == r"\&");
+}
 
 impl TypeDarwin {
     fn groff_replace_escapes(line: &str) -> String {
-        unimplemented!()
+        line.replace(".Nm", "CMDNAME")
```

Comment:
Can you remove the commented python stub when adding a function? It makes reviewing easier